## Representation and Reporting - D210 Task 1 PA
Vincent Taylor

Part I (Interactive Dashboard)

***A1: Copy of Dashboard***

For the creation of this dashboard I paired the exisiting churn dataset alongside some general telecom data pulled from Kaggle the link to this dashboard can be accessed at this link (https://public.tableau.com/app/profile/vincent.taylor1660/viz/D210Task1_17041171570210/D210?publish=yes).

Once I was able to find a dataset to use alongside the churn data, the data will need to be cleaned and I will need to make sure that there are uniform columns for both datasets in order to visualize the data in Tableau. I will also need to remove unecessary columns from the Kaggle dataset as well as the churn clean dataset. In order to perform the data cleaning for both of the spreadsheets I will accomplish this by using prior code from a past PA. 

***<u> Links to both csv files; </u>***

- churn_clean.csv: (https://access.wgu.edu/ASP3/aap/content/f9tjr8djg83jd8c3sdf8.zip)
- Kaggle Telecom Dataset: (https://www.kaggle.com/code/bhartiprasad17/customer-churn-prediction/data)

---

#### Data Cleaning

To begin we will start by cleaning the data from the churn_clean file and then we will clean the data from the kaggle dataset. After this we will import the dataset into Tableau.

---

In [1]:
#Cleaning Churn dataset

#Import Libraries
import pandas as pd
import numpy as np

# import data from csv
df = pd.read_csv('churn_clean.csv', index_col=0)
df.head()

Customer_id                           Interaction  \
CaseOrder                                                     
1             K409198  aa90260b-4141-4a24-8e36-b04ce1f4f77b   
2             S120509  fb76459f-c047-4a9d-8af9-e0f7d4ac2524   
3             K191035  344d114c-3736-4be5-98f7-c72c281e2d35   
4              D90850  abfa2b40-2d43-4994-b15a-989b8c79e311   
5             K662701  68a861fd-0d20-4e51-a587-8a90407ee574   

                                        UID         City State  \
CaseOrder                                                        
1          e885b299883d4f9fb18e39c75155d990  Point Baker    AK   
2          f2de8bef964785f41a2959829830fb8a  West Branch    MI   
3          f1784cfa9f6d92ae816197eb175d3c71      Yamhill    OR   
4          dc8a365077241bb5cd5ccd305136b05e      Del Mar    CA   
5          aabb64a116e83fdc4befc1fbab1663f9    Needville    TX   

                          County    Zip       Lat        Lng  Population  ...  \
CaseOrder                                                                 ...   
1          Prince of Wales-Hyder  99927  56.25100 -133.37571          38  ...   
2                         Ogemaw  48661  44.32893  -84.24080       10446  ...   
3                        Yamhill  97148  45.35589 -123.24657        3735  ...   
4                      San Diego  92014  32.96687 -117.24798       13863  ...   
5                      Fort Bend  77461  29.38012  -95.80673       11352  ...   

          MonthlyCharge Bandwidth_GB_Year Item1  Item2  Item3  Item4 Item5  \
CaseOrder                                                                    
1            172.455519        904.536110     5      5      5      3     4   
2            242.632554        800.982766     3      4      3      3     4   
3            159.947583       2054.706961     4      4      2      4     4   
4            119.956840       2164.579412     4      4      4      2     5   
5            149.948316        271.493436     4      4      4      3     4   

          Item6 Item7  Item8  
CaseOrder                     
1             4     3      4  
2             3     4      4  
3             3     3      3  
4             4     3      3  
5             4     4      5  

[5 rows x 49 columns]

In [2]:
#Data Cleaning

#renaming columns to python casing.
col_head = {
    'CaseOrder': 'case_order',
    'Customer_id': 'customer_id',
    'Interaction': 'interaction',
    'City': 'city',
    'State': 'state',
    'County': 'county',
    'Zip': 'zip',
    'Lat': 'lat',
    'Lng': 'lng',
    'Population': 'population',  
    'Area': 'area',  
    'Job': 'job',
    'Children': 'children',
    'Age': 'age', 
    'Income': 'income',  
    'Marital': 'married',
    'Gender': 'gender',
    'Churn': 'churn',
    'Outage_sec_perweek': 'outage_sec_perweek',
    'Email': 'email',
    'Contacts': 'contacts',
    'Yearly_equip_failure': 'yearly_equip_failure',
    'Techie': 'techie',
    'Contract': 'contract',
    'Port_modem': 'port_modem',
    'Tablet': 'tablet',
    'InternetService': 'internet_service',
    'Phone': 'phone',
    'Multiple': 'multiple_lines',
    'OnlineSecurity': 'online_security',
    'OnlineBackup': 'online_backup',
    'DeviceProtection': 'device_protection',
    'TechSupport': 'tech_support',
    'StreamingTV': 'streaming_tv',
    'StreamingMovies': 'streaming_movies',
    'PaperlessBilling': 'paperless_billing',
    'PaymentMethod': 'payment_method',
    'Tenure': 'tenure',
    'MonthlyCharge': 'monthly_charge',
    'Bandwidth_GB_Year': 'bandwidth_gb_year',
    'Item1': 'timely_response',
    'Item2': 'timely_fixes',
    'Item3': 'timely_replacements',
    'Item4': 'reliability',
    'Item5': 'options',
    'Item6': 'respectful_response',
    'Item7': 'courteous_exchange',
    'Item8': 'active_listening'}
df.rename(columns=col_head, inplace=True)

#Reexpressing the values of payment method to be more concise.
dict_pay = {"payment_method": {
     "Credit Card (automatic)": "Autopay",
     "Bank Transfer(automatic)": "Autopay",
     "Electronic Check": "eCheck",
     "Mailed Check": "Check",
}}
df.replace(dict_pay, inplace = True)

#Rounding monthly_charge column to be more concise
df['monthly_charge'] = df.monthly_charge.round(2)

#Rounding tenure column to be more concise
df['tenure'] = df.tenure.round(0)

#Reexpressing the values of marital to be more concise.
dict_mar = {"married": {
     "Never Married": "No",
     "Divorced": "No",
     "Widowed": "No",
     "Single": "No",
     "Married": "Yes",
     "Separated": "No",    
}}
df.replace(dict_mar, inplace = True)

#Reexpressing the values of payment method to be more concise.
dict_contract = {"contract": {
     "Two Year": "Two year",
}}
df.replace(dict_contract, inplace = True)

# Updating children to yes/no
def convert_to_yes_no(value):
    if value > 0:  
        return 'Yes'
    else:
        return 'No'

df['dependent'] = df['children'].apply(convert_to_yes_no)

# Dropping the original 'children' column
df.drop(columns=['children'], inplace=True)

#Changing the following to category data type.
df_cat = ['area','married',
          'gender','payment_method','timely_response','timely_fixes',
          'timely_replacements','reliability','options','respectful_response'
          ,'courteous_exchange','active_listening'] 
df[df_cat] = df[df_cat].astype("category")

df.head()

customer_id                           interaction  \
CaseOrder                                                     
1             K409198  aa90260b-4141-4a24-8e36-b04ce1f4f77b   
2             S120509  fb76459f-c047-4a9d-8af9-e0f7d4ac2524   
3             K191035  344d114c-3736-4be5-98f7-c72c281e2d35   
4              D90850  abfa2b40-2d43-4994-b15a-989b8c79e311   
5             K662701  68a861fd-0d20-4e51-a587-8a90407ee574   

                                        UID         city state  \
CaseOrder                                                        
1          e885b299883d4f9fb18e39c75155d990  Point Baker    AK   
2          f2de8bef964785f41a2959829830fb8a  West Branch    MI   
3          f1784cfa9f6d92ae816197eb175d3c71      Yamhill    OR   
4          dc8a365077241bb5cd5ccd305136b05e      Del Mar    CA   
5          aabb64a116e83fdc4befc1fbab1663f9    Needville    TX   

                          county    zip       lat        lng  population  ...  \
CaseOrder                                                                 ...   
1          Prince of Wales-Hyder  99927  56.25100 -133.37571          38  ...   
2                         Ogemaw  48661  44.32893  -84.24080       10446  ...   
3                        Yamhill  97148  45.35589 -123.24657        3735  ...   
4                      San Diego  92014  32.96687 -117.24798       13863  ...   
5                      Fort Bend  77461  29.38012  -95.80673       11352  ...   

          bandwidth_gb_year timely_response timely_fixes  timely_replacements  \
CaseOrder                                                                       
1                904.536110               5            5                    5   
2                800.982766               3            4                    3   
3               2054.706961               4            4                    2   
4               2164.579412               4            4                    4   
5                271.493436               4            4                    4   

           reliability options respectful_response courteous_exchange  \
CaseOrder                                                               
1                    3       4                   4                  3   
2                    3       4                   3                  4   
3                    4       4                   3                  3   
4                    2       5                   4                  3   
5                    3       4                   4                  4   

           active_listening  dependent  
CaseOrder                               
1                         4         No  
2                         4        Yes  
3                         3        Yes  
4                         3        Yes  
5                         5         No  

[5 rows x 49 columns]

In [3]:
#Adding column to dataframe for tableau use
df['dataset'] = 'WGU Telecom Co.'

In [4]:
#Cleaning Kaggle Dataset

# import data from csv
kaggle_df = pd.read_csv('kaggle_churn.csv', index_col=0)
kaggle_df.head()

gender  SeniorCitizen Partner Dependents  tenure PhoneService  \
customerID                                                                  
7590-VHVEG  Female              0     Yes         No       1           No   
5575-GNVDE    Male              0      No         No      34          Yes   
3668-QPYBK    Male              0      No         No       2          Yes   
7795-CFOCW    Male              0      No         No      45           No   
9237-HQITU  Female              0      No         No       2          Yes   

               MultipleLines InternetService OnlineSecurity OnlineBackup  \
customerID                                                                 
7590-VHVEG  No phone service             DSL             No          Yes   
5575-GNVDE                No             DSL            Yes           No   
3668-QPYBK                No             DSL            Yes          Yes   
7795-CFOCW  No phone service             DSL            Yes           No   
9237-HQITU                No     Fiber optic             No           No   

           DeviceProtection TechSupport StreamingTV StreamingMovies  \
customerID                                                            
7590-VHVEG               No          No          No              No   
5575-GNVDE              Yes          No          No              No   
3668-QPYBK               No          No          No              No   
7795-CFOCW              Yes         Yes          No              No   
9237-HQITU               No          No          No              No   

                  Contract PaperlessBilling              PaymentMethod  \
customerID                                                               
7590-VHVEG  Month-to-month              Yes           Electronic check   
5575-GNVDE        One year               No               Mailed check   
3668-QPYBK  Month-to-month              Yes               Mailed check   
7795-CFOCW        One year               No  Bank transfer (automatic)   
9237-HQITU  Month-to-month              Yes           Electronic check   

            MonthlyCharges TotalCharges Churn  
customerID                                     
7590-VHVEG           29.85        29.85    No  
5575-GNVDE           56.95       1889.5    No  
3668-QPYBK           53.85       108.15   Yes  
7795-CFOCW           42.30      1840.75    No  
9237-HQITU           70.70       151.65   Yes

In [5]:
kaggle_df.info()

<class 'pandas.core.frame.DataFrame'>
Index: 7043 entries, 7590-VHVEG to 3186-AJIEK
Data columns (total 20 columns):
 #   Column            Non-Null Count  Dtype  
---  ------            --------------  -----  
 0   gender            7043 non-null   object 
 1   SeniorCitizen     7043 non-null   int64  
 2   Partner           7043 non-null   object 
 3   Dependents        7043 non-null   object 
 4   tenure            7043 non-null   int64  
 5   PhoneService      7043 non-null   object 
 6   MultipleLines     7043 non-null   object 
 7   InternetService   7043 non-null   object 
 8   OnlineSecurity    7043 non-null   object 
 9   OnlineBackup      7043 non-null   object 
 10  DeviceProtection  7043 non-null   object 
 11  TechSupport       7043 non-null   object 
 12  StreamingTV       7043 non-null   object 
 13  StreamingMovies   7043 non-null   object 
 14  Contract          7043 non-null   object 
 15  PaperlessBilling  7043 non-null   object 
 16  PaymentMethod     7043 non-null 

In [6]:
# Renaming columns to Python casing
col_head = {
    'customerID': 'customer_id',
    'SeniorCitizen': 'senior_citizen',
    'Partner': 'married',
    'Dependents': 'dependent',
    'PhoneService': 'phone',
    'MultipleLines': 'multiple_lines',
    'OnlineSecurity': 'online_security',
    'OnlineBackup': 'online_backup',
    'DeviceProtection': 'device_protection',
    'TechSupport': 'tech_support',
    'StreamingTV': 'streaming_tv',
    'StreamingMovies': 'streaming_movies',
    'PaperlessBilling': 'paperless_billing',
    'PaymentMethod': 'payment_method',
    'Contract': 'contract',
    'Churn': 'churn',
    'InternetService': 'internet_service',
    'TotalCharges': 'total_charges',
    'MonthlyCharges': 'monthly_charge',
    'gender': 'gender'
}

# Renaming columns in the DataFrame
kaggle_df.rename(columns=col_head, inplace=True)

# Reexpressing the values of payment method to be more concise
kaggle_pay = {
    "payment_method": {
        "Credit card (automatic)": "Autopay",
        "Bank transfer (automatic)": "Autopay",
        "Electronic check": "eCheck",
        "Mailed check": "Check",
    }
}
kaggle_df.replace(kaggle_pay, inplace=True)

# Reexpressing the values of internet_service to be more concise
kaggle_int = {
    "internet_service": {
        "No": "None",
        "Fiber optic": "Fiber Optic"
    }
}
kaggle_df.replace(kaggle_int, inplace=True)

kaggle_no_internet = {
    "online_security": {"No internet service": "No"},
    "online_backup": {"No internet service": "No"},
    "device_protection": {"No internet service": "No"},
    "tech_support": {"No internet service": "No"},
    "streaming_tv": {"No internet service": "No"},
    "streaming_movies": {"No internet service": "No"}
}

# Renaming columns in the DataFrame
kaggle_df.replace(kaggle_no_internet, inplace=True)

# Reexpressing the values of gender to be more concise
kaggle_lines = {
    "multiple_lines": {
        "No phone service": "No",
    }
}
kaggle_df.replace(kaggle_lines, inplace=True)

# Displaying the modified DataFrame
kaggle_df.head()

gender  senior_citizen married dependent  tenure phone  \
customerID                                                           
7590-VHVEG  Female               0     Yes        No       1    No   
5575-GNVDE    Male               0      No        No      34   Yes   
3668-QPYBK    Male               0      No        No       2   Yes   
7795-CFOCW    Male               0      No        No      45    No   
9237-HQITU  Female               0      No        No       2   Yes   

           multiple_lines internet_service online_security online_backup  \
customerID                                                                 
7590-VHVEG             No              DSL              No           Yes   
5575-GNVDE             No              DSL             Yes            No   
3668-QPYBK             No              DSL             Yes           Yes   
7795-CFOCW             No              DSL             Yes            No   
9237-HQITU             No      Fiber Optic              No            No   

           device_protection tech_support streaming_tv streaming_movies  \
customerID                                                                
7590-VHVEG                No           No           No               No   
5575-GNVDE               Yes           No           No               No   
3668-QPYBK                No           No           No               No   
7795-CFOCW               Yes          Yes           No               No   
9237-HQITU                No           No           No               No   

                  contract paperless_billing payment_method  monthly_charge  \
customerID                                                                    
7590-VHVEG  Month-to-month               Yes         eCheck           29.85   
5575-GNVDE        One year                No          Check           56.95   
3668-QPYBK  Month-to-month               Yes          Check           53.85   
7795-CFOCW        One year                No        Autopay           42.30   
9237-HQITU  Month-to-month               Yes         eCheck           70.70   

           total_charges churn  
customerID                      
7590-VHVEG         29.85    No  
5575-GNVDE        1889.5    No  
3668-QPYBK        108.15   Yes  
7795-CFOCW       1840.75    No  
9237-HQITU        151.65   Yes

In [7]:
kaggle_df['dataset'] = 'Competitor'

In [8]:
#Checking for null values
if kaggle_df.isnull().sum().sum() == 0:
    print("No null values.")
else:
    print("Null values exist!")

No null values.


In [9]:
# dropping columns not relevant to analysis (Kaggle)
kaggle_df = kaggle_df.drop(columns=['senior_citizen','total_charges'])

# dropping columns not relevant to analysis (churn_clean)
df = df.drop(columns=['interaction', 'UID', 'TimeZone', 'job', 'age', 'income',
                       'outage_sec_perweek', 'email',
                       'contacts', 'yearly_equip_failure', 'techie', 'port_modem',
                       'tablet','bandwidth_gb_year','timely_response', 'timely_fixes', 
                       'timely_replacements', 'reliability',
                       'options', 'respectful_response', 'courteous_exchange',
                       'active_listening','customer_id', 'city', 'state', 'county', 'zip', 'lat', 'lng',
       'population', 'area'])

In [11]:
# Consolidating both datasets
combined_df = pd.concat([df, kaggle_df], ignore_index=True)

#Saving dataset to csv
combined_df.to_csv('combined_df.csv', index=False)
combined_df

married  gender churn        contract internet_service phone  \
1               No    Male    No        One year      Fiber Optic   Yes   
2              Yes  Female   Yes  Month-to-month      Fiber Optic   Yes   
3               No  Female    No        Two year              DSL   Yes   
4              Yes    Male    No        Two year              DSL   Yes   
5               No    Male   Yes  Month-to-month      Fiber Optic    No   
...            ...     ...   ...             ...              ...   ...   
6840-RESVB     Yes    Male    No        One year              DSL   Yes   
2234-XADUH     Yes  Female    No        One year      Fiber Optic   Yes   
4801-JZAZL     Yes  Female    No  Month-to-month              DSL    No   
8361-LTMKD     Yes    Male   Yes  Month-to-month      Fiber Optic   Yes   
3186-AJIEK      No    Male    No        Two year      Fiber Optic   Yes   

           multiple_lines online_security online_backup device_protection  \
1                      No             Yes           Yes                No   
2                     Yes             Yes            No                No   
3                     Yes              No            No                No   
4                      No             Yes            No                No   
5                      No              No            No                No   
...                   ...             ...           ...               ...   
6840-RESVB            Yes             Yes            No               Yes   
2234-XADUH            Yes              No           Yes               Yes   
4801-JZAZL             No             Yes            No                No   
8361-LTMKD            Yes              No            No                No   
3186-AJIEK             No             Yes            No               Yes   

           tech_support streaming_tv streaming_movies paperless_billing  \
1                    No           No              Yes               Yes   
2                    No          Yes              Yes               Yes   
3                    No           No              Yes               Yes   
4                    No          Yes               No               Yes   
5                   Yes          Yes               No                No   
...                 ...          ...              ...               ...   
6840-RESVB          Yes          Yes              Yes               Yes   
2234-XADUH           No          Yes              Yes               Yes   
4801-JZAZL           No           No               No               Yes   
8361-LTMKD           No           No               No               Yes   
3186-AJIEK          Yes          Yes              Yes               Yes   

           payment_method  tenure  monthly_charge dependent          dataset  
1                 Autopay     7.0          172.46        No  WGU Telecom Co.  
2                 Autopay     1.0          242.63       Yes  WGU Telecom Co.  
3                 Autopay    16.0          159.95       Yes  WGU Telecom Co.  
4                   Check    17.0          119.96       Yes  WGU Telecom Co.  
5                   Check     2.0          149.95        No  WGU Telecom Co.  
...                   ...     ...             ...       ...              ...  
6840-RESVB          Check    24.0           84.80       Yes       Competitor  
2234-XADUH        Autopay    72.0          103.20       Yes       Competitor  
4801-JZAZL         eCheck    11.0           29.60       Yes       Competitor  
8361-LTMKD          Check     4.0           74.40        No       Competitor  
3186-AJIEK        Autopay    66.0          105.65        No       Competitor  

[17043 rows x 19 columns]

***(A2) Dashboard Installation Instructions***

To Install this dashboard in Tableau follow the instructions below. Additionally if you would not like to install this dashboard please use this link to use the Tableau Public dashboard (https://public.tableau.com/app/profile/vincent.taylor1660/viz/D210Task1_17041171570210/D210?publish=yes).


Method one: 

    1. Open Tableau and select File > Open
    2. Select my D210 Task1 Zip file > and Open the .twb file.
    3. Once imported you will be prompted to upload both the churn_clean.csv and the kaggle_df.csv
    4. Upload both of these files and from there the data should work as normal. 
    5. If you need to view the worksheets you can right click on a tab and select unhide all sheets. 
    
    
Method two: 

<u>Steps for Creating Worksheet</u>

        1. Open Tableau on your local machine. 
        2. Once it has been opened select file > open
        3. Select combined_df.csv file from there it should load you onto the datasource tab.
        4. Select Sheet 1
        5. Rename sheet 1 to "Count of Records"
        6. In Tables Select "Dataset" from the dimensions area and move it to the rows.
        7. Select combined_df.csv(Count) and move that to the Text marks.
        8. Hover over the field label for rows and right click and select hide. 
        9. Change Fit to entire view.
       10. Create a new Worksheet and name it "Monthly Charge by Contract".
       11. In tables move Contract (Dimensons) to Rows and Monthly Charge to the text marks tab. 
       12. From the label tab click on monthly charge and and change the measure to average.
       13. Change Fit to entire view.
       14. Create a new worksheet and rename it to "Average Tenure by Contract"
       15. Select Tenure and move it to both the label and size marks, and move contract to the text box.
       16. In the top right corner move to select Show Me and select the Tree Map.
       17. Change fit to entire view.
       18. Create a new worksheet and name it "Gender Breakdown".
       19. Move Contract to columns.
       20. Select Gender and move it to Detail, Color, and label tooltips.
       21. Select "Show Me" and select the pie chart.
       22. Change Fit to Entire view.
       23. Select the label tool tip for Gender and under the quick calculations select % of Total.
       24  Select the Label tool tip and check the box "Show Tick Marks"
       25. Create another worksheet and this time name it "Trailing Twelve Month Churn"
       26. Move Churn to Rows and Combined_df (Count) to the label tooltip and make sure that text table is the default type.
       27. Change fit to entire view.
       28. Create one worksheet and name this one "Average Tenure Count"
       29. Move combined_df Count to both colors and Rows.
       30. From the measures tab right click on Tenure > Create > Bins > Select Bin size of 5.
       31. Move Tenure(bin) from the dimensions tab to the columns. 
       32. Move Tenure(bin) to the Filter as well as the dataset to the filter.
       33. Visualize as a barchart and change the fit to the entire view.
       
<u>Steps for Creating Dashboard</u>

        1. Select new Dashboard.
        2. Name this dashboard "KPI Contract Data"
        3. Move the following sheets and organize to your liking. Count of Records, Monthly Charge by Contract, Average Tenure by Contract, Average Tenure Count, and Gender.
        4. Select the filter for each worksheet to ensure that it updates as items on the dashboard are selected.
        5. Change the size to Automatic.
        6. Create a new Dashboard and name this one Consumer Products. 
        7. Place the remaining items on this dashboard.
        8. Select New Story and Select the blank in the left side of the screen and bring in both of the Dashboards. 

### Navigating the Dashboard

(A3): 

To navigate the dashboard select the tab D210 it will take you to the story which contains four tabs. Below is a brief description for each of the tabs.

Introduction: Just a title page for giving a description of what this analysis will be about, similar to a slide deck in corporate settings.

Customer Contract Data: This is a list of various KPIs as contracts would affect our bottom-line and to also be able to pull insights from whichever dataset we would like to see. It can be filtered by clicking on any portion of the dashboard and it will also update the other dashboard items. Alternatively there is  a slider for monthly charge and as well as selecting both the WGU telecom Co or the competitor box in order to see how each demographic performs.

Consumer Products: This tab contains various KPIs as it relates to payment methods and other product offerings available through both companies. This as well can be filtered by selecting anywhere on the dashboard or using the filter items in the container to the right.

Reccomendations: This tab has a few key takeaways that I would provide to the executives in order to give them more insight on what seems to be leading the numbers.

#### Part II (Storytelling with data)

Panopto: 

### Part III (Reflection Paper)

C1: The purpose of this dashboard was to be able to provide context for internal stakeholders as it pertrained to customer statistics vs our competitiors. Over the past few assignments we have been analyzing our internal data but still have not comapred our data to an external dataset to see where we stand against our competitors. The dashboards are functioning as our comparison to the industry. In my inital search for a dataset to use in this analysis finding another set of data that had matching columns to our churn_clean and matched our own data dictionary was my primary objective. I was able to find one and really wanted to make sure that I was able to do an Apples to Apples comparison, so I made sure to remove rows that would were not included on both. For example instead of an age column our external dataset had wether a customer was a senior citizen or not. There was also survey satisfaction data that was included on the churn_clean.csv and not on the other so those columns were omitted for that reason. 

C2: The variables in the additional dataset enhance our insights due to the similarity in the responses I saw in the rows. The external dataset had many of the same products that our company offers; (ie, OnlineSecurity, OnlineBackup) so I thought this would be a great dataset to use. One of the downsides to this dataset is that it only contained 7043 unique responses so certain numbers may be skewed and could potentially tell a story that is not true, so I had to be very careful in the data insights I did choose to use in my recommendations.

C3: One insight that executives can use to enhance insights would definitely be the customer payment types. I work in FinTech and this is a deliverable I am responsible for each quarter as we try to dissuade customers from using slower payment methods such as checks. An executive would be able to use this specific representation to judge where we stand in terms of getting slower paying customers to opt for electronic methods as well as if our competitors are running into the same issue. Another key insight would be the monthly charge text table found on the Customer Contract data. Even with the skewed data our customers pay twice as much to retain services through us compared to the competitor data. On average a higher majority of our customers tend to be subscribed to more than one item compared to our competitors which could explain this, but it would defintiely be something I would recommed that senior management look into.

C4: On the Customer Contract data tab a user is able to modify the datasets to select between competitor vs WGU telecom co. This would provide two seperate looks with only data from each of the respective companies being considered for a more straight forward analysis. Moreover, on the Consumer Products tab users are able to select different internet types in order to see how various payment methods compare. Lastly a user has the ability to click on any dashboard if they are looking for insights comapred to a certain product and will get a display focused on that specific insight. Users can also select the escape if they would like to revert back to the original version of the dashboard.

C5: In order to ensure that colorblind users are able to also use this dashboard I made sure to use the color blind palette when I was doing various graphs, which is why all of the dataset graphs are the same color.

C6: One of the stories I wanted to be able to tell is that we have more customers who pay by check than our competitors. Looking at the dataset 23% of our customers pay by check which for comaprison would put us in line with our competitors who check represents 23% of their payment method as well, with the key difference of almost 3000 less responses. This would lead me to recommend to senior management that we would need to have our representatives continue to push an electronic method. Another representation I wanted to tell is on the Customer Contract Data is being able to use the monthly charge each month to back into how likely a customer is to churn selecting any charge on there will show you how often someone churned in the past tweleve months based on their contract type. This is another important KPI for senior management in order to figure out which demographic is most likely to churn based on their contract.

C7: The audience I used for this analysis was a senior management team, that would be focused on items that move the needle, due to this I made sure to keep the recommendations short in order to leave time for questions that would be had by the senior management. Alternatively working in finance I have to deal with making sure that I talk about items that lead to a material impact. This led to me focusing on more subscription based services we offer. I would adapt this if it was for a specific department in order to make sure that the items being discussed are things that they deal with in their day to day.

C8: Using a colorblind palette allows for everyone to be able to view the same color spectrum, moreover the panopto video will allow for people who are blind or have vision impairments to also understand what is going on. Lastly I have uploaded this item on a public website to allow all audiences to access and engage with the data analysis tools free of charge.

C9: I wanted to ensure that I was able to keep the attention of senior management as they are busy with multiple things throughout their day so I made sure to keep topics brief and focused on the bottom line which impacts the company directly. I also added in visuals to ensure alongside the brief storytelling to assist with being able to understand some of the more complex insights. 

#### Sources 
CUSTOMER CHURN PREDICTION 📈. (n.d.). Kaggle.com. https://www.kaggle.com/code/bhartiprasad17/customer-churn-prediction/data


#### References
 No sources/third party information were used in the creation of this presentation.

#### Link to Panopto Recording

Link: https://wgu.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=0b0fcc19-8e1f-4c0d-8d48-b0ea0028040d